calibration/CTDMOG/CGINS-CTDMOG-13617__20150618.csv

```diff
@@ -20,6 +20,6 @@ serial,name,value,notes
 37-13617,CC_ptcb1,5e-05,
 37-13617,CC_ptcb2,0,
 37-13617,CC_ptempa0,201.1284,
-37-13617,CC_ptempa1,-0.07966973,
+37-13617,CC_ptempa1,-7.966974e-02,
 37-13617,CC_ptempa2,1.454982e-06,
-37-13617,CC_wbotc,3.4781e-07,
+37-13617,CC_wbotc,3.478056e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT